src/main/java/com/google/api/codegen/ruby/RubyProvider.java

```diff
@@ -38,24 +31,6 @@
   static final String SNIPPET_RESOURCE_ROOT =
       RubyGapicProvider.class.getPackage().getName().replace('.', '/');
 
-  public <Element> void output(
-      String packageRoot, String outputPath, Multimap<Element, GeneratedResult> elements)
-      throws IOException {
-    Map<String, Doc> files = new LinkedHashMap<>();
-    for (Map.Entry<Element, GeneratedResult> entry : elements.entries()) {
-      Element element = entry.getKey();
-      GeneratedResult generatedResult = entry.getValue();
-      String root;
-      if (element instanceof Method) {
-        root = ((Method) element).getFile().getFullName().replace('.', '/');
```

Comment:
Actually, I guess this would only be used by a Method-view codegen process, and there isn't one of those for Ruby?